src/lib/isEmail.js

```diff
@@ -24,10 +25,16 @@ export default function isEmail(str, options) {
   assertString(str);
   options = merge(options, default_email_options);
 
+  if (options.require_display_name) {
+    options.allow_display_name = true;
+  }
+
   if (options.allow_display_name) {
```

Comment:
Happy to make that change. I wasn't sure if the option was in use elsewhere, and wanted to avoid the case where someone had explicitly erroneously set `allow_display_name` to `false`, while setting `require_display_name` to `true`. Looking at the code, that doesn't seem to be a concern.